### PR TITLE
Release 2.9.4

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming v2.9.4 production release? Well you've just found it, congratulations! :tada:

This is based on development of which the latest commit is the merge of v2.9.4-beta4. (This is based on that beta)

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
-- Nothing worth of mentioning
- [x] Verify that all feature flags are flipped appropriately
image

- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
-- No new metrics

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+      "[Added] Add syntax mapping for HAML - #13009. Thanks @alexanderadam!",
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Added] Add a menu item to view a committed file change on GitHub - #12492. Thanks @haykam821!",
+      "[Fixed] Diffs are scrolled to the top when switching between files - #12980",
+      "[Fixed] Fix SSH prompt for unknown hosts in some scenarios - #13050",
+      "[Fixed] Apply syntax highlighting when viewing a small change for the first time - #13004",
+      "[Fixed] Wrap long email addresses in the misattributed commit warning popover - #13044",
+      "[Fixed] Refresh diffs when application receives focus - #12962",
+      "[Fixed] Tokens are not considered invalid when accessing GitHub Enterprise from outside of the company's VPN - #12943",
+      "[Fixed] Show \"add repo\" dialog when opening repo from CLI - #12935",
```
Comment (sergiou87) [inline changelog.json:13]: This one was marked as `no-notes` in the original PR, probably not worth adding?
```suggestion
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+      "[Fixed] Continue merge flow after merge conflicts are resolved with exterior conflict tool - #12881",
```
Comment (sergiou87) [inline changelog.json]: Not sure if `exterior` is correct here, but `external` sounds much more familiar to me 😳  Feel free to ignore this comment 👀 
```suggestion
      "[Fixed] Continue merge flow after merge conflicts are resolved with external conflict tool - #12881",
```

File: changelog.json (hunk 1 above)
Comment (tidy-dev) [reply changelog.json:13]: This is the one that was the recent bug bounty... Seems we may want it for that visibility?

File: changelog.json (hunk 2 above)
Comment (tidy-dev) [reply changelog.json]: :D Your right! It probably translated over in my head from working on houses.. exterior vs interior work...

File: changelog.json (hunk 1 above)
Comment (billygriffin) [reply changelog.json:13]: I'm fine with keeping it - it does represent actual functionality.

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+      "[Added] Add a menu item to view a committed file change on GitHub - #12492. Thanks @haykam821!",
+      "[Fixed] Diffs are scrolled to the top when switching between files - #12980",
+      "[Fixed] Fix SSH prompt for unknown hosts in some scenarios - #13050",
+      "[Fixed] Apply syntax highlighting when viewing a small change for the first time - #13004",
+      "[Fixed] Wrap long email addresses in the misattributed commit warning popover - #13044",
+      "[Fixed] Refresh diffs when application receives focus - #12962",
+      "[Fixed] Tokens are not considered invalid when accessing GitHub Enterprise from outside of the company's VPN - #12943",
+      "[Fixed] Show \"add repo\" dialog when opening repo from CLI - #12935",
+      "[Fixed] Continue merge flow after merge conflicts are resolved with external conflict tool - #12881",
+      "[Fixed] App no longer crashes on successful merge after conflicts resolved - #12888",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Prevent crash on successful merge after conflicts resolved - #12888",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,28 @@
...
+    "2.9.4": [
+      "[Added] Add syntax mapping for HAML - #13009. Thanks @alexanderadam!",
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Added] Add a menu item to view a committed file change on GitHub - #12492. Thanks @haykam821!",
+      "[Fixed] Diffs are scrolled to the top when switching between files - #12980",
+      "[Fixed] Fix SSH prompt for unknown hosts in some scenarios - #13050",
+      "[Fixed] Apply syntax highlighting when viewing a small change for the first time - #13004",
+      "[Fixed] Wrap long email addresses in the misattributed commit warning popover - #13044",
+      "[Fixed] Refresh diffs when application receives focus - #12962",
+      "[Fixed] Tokens are not considered invalid when accessing GitHub Enterprise from outside of the company's VPN - #12943",
```
Comment (billygriffin) [inline changelog.json]: I'm not wedded to this, feel free to reject or reword if you don't like this.
```suggestion
      "[Fixed] Only consider tokens invalid when 401 error comes from GitHub Enterprise - #12943",
```

**Review by billygriffin — COMMENTED** [review]
Very minor changes - I'm going to go ahead and accept and then approve since the release is tomorrow.

**Review by billygriffin — APPROVED** [review]
🚀

**Comment by wmgl5593** [comment]
gh pr checkout 13065[gh pr checkout 13065]()

**Pull request merged**